src/CMakeLists.txt

```diff
@@ -10,6 +10,11 @@ add_executable(sfml_cuda
 # )
 # set_target_properties(sfml_cuda PROPERTIES CUDA_SEPARABLE_COMPILATION ON)
 
+find_package(OpenMP)
```

Comment:
Keep this line in the main CMakeLists file, since this one is just supposed to do linking and stuff, not search for new features.